ViewGeneratorCore/tools/webpack.config.js

```diff
@@ -0,0 +1,44 @@
+﻿const glob = require('glob');
```

Comment:
You should convert this file into a ts

Reply:
Will do, if webpack does not complain. Same for the another configuration file that I've added in ReactViewResources project

Reply:
Converted